fetch.bs

```diff
@@ -791,7 +792,7 @@ the empty string,
    <td><code>child-src</code>
    <td>HTML's <code>&lt;iframe></code> and <code>&lt;frame></code>
   <tr>
-   <td>""
+   <td>"" or "<code>fetch</code>"
```

Comment:
I think `("<code>fetch</code>")` would be clearer since you can only use it as input, right?